workers/unity/Assets/Playground/Scripts/Camera/InitCameraSystem.cs

```diff
@@ -37,6 +37,14 @@ protected override void OnUpdate()
 
                 Cursor.lockState = CursorLockMode.Locked;
             }
+
+            // Disable system after first run.
+            Enabled = false;
```

Comment:
This isn't related to PR, but I think this was missing here. The code in the system isn't really intended to be ran multiple times

Reply:
this might disable the system before it is actually run depending on when the components are added to the entities

Reply:
Good point. This should be fixed if moved to the cycle above, right?

Reply:
Yeah if its in the `for` loop it should be good 👍